app/javascript/components/artist_works.js

```diff
@@ -30,6 +46,13 @@ class ArtistWorks extends React.Component {
             <p>{work.media}</p>
           </div>
         )}
+
+        <form onSubmit = {this.handleSubmit} name="commissionsForm">
+          <textarea type="TEXT" name="comment" id="comment" value={this.state.comment}
```

Comment:
Separate the `textarea` component by putting each prop on a newline so it is more readable